core/domain/feedback_jobs_one_off_test.py

```diff
@@ -166,7 +166,7 @@ class FeedbackSubjectOneOffJobTest(test_utils.GenericTestBase):
     }
 
     EXPECTED_THREAD_DICT7 = {
-        'text': u'â, ??î or ôu🕧� n☁i✑💴++$-💯 ♓!🇪🚑🌚‼⁉4⃣od; /⏬®;😁☕😁:☝)😁😁😍1!@#',
```

Comment:
@seanlip:  Are we talking about this line? It comprises of 79 characters. I checked it locally too. We have an 80 characters limit at present.  
Please correct me if I am considering the wrong line.  
Thanks!

Reply:
Yes, that line. Weird, it shows as 81 chars when I paste it in Sublime text. Maybe some of these unicode chars are being counted as 2 rather than 1 char?

/cc @brianrodri

Reply:
Yes. I think so too. I confirmed the character check is working properly by increasing the characters to 81. The error message then pops up saying, "Line too long". So the lint check is working.

Reply:
Hm, ok. Might just be a unicode thing working differently on different machines, then.

Thank you very much for checking!

Reply:
No problem, @seanlip! Thanks :)